### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by maks-golovatyuk**
https://maks-golovatyuk.github.io/layout_moyo-header/
https://maks-golovatyuk.github.io/layout_moyo-header/report/html_report/

### CODE REVIEW

**Review by liliia-livandovskaya — CHANGES_REQUESTED** [review]
Pay attention to the tests. They show you the difference between your code and the mockup.
Screenshot 2022-10-06 at 12 12 40

File: src/index.html [hunk 1]
```diff
@@ -5,10 +5,67 @@
...
+          src="images/logo.png"
+          alt="Moyo logo"
+          class="logo"
+        >
+      </div>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__li">
+            <a href="*" class="nav__link is-active">
```
Comment (Anastasiia-Svintsova) [inline src/index.html:24]: all your links lead to 404 page( Rewrite href atribute in all links
```suggestion
            <a href="#Apple" class="nav__link is-active">
```

File: src/style.css [hunk 2]
```diff
@@ -1 +1,82 @@
...
+
+.nav {
+  background-color: white;
+}
+
+.nav__list {
+  display: flex;
+  flex-direction: row;
+  margin: 0;
+  padding: 0;
```
Comment (Anastasiia-Svintsova) [inline src/style.css:40]: delete this, it's useless here

**Review by Anastasiia-Svintsova — CHANGES_REQUESTED** [review]
Good job! Just fix links issues

**Review by Anastasiia-Svintsova — CHANGES_REQUESTED** [review]
Links still lead to 404 page. did you deploy after the changes?

**Review by qoa11a — CHANGES_REQUESTED** [review]
I see the same problems that were mentioned in previous comments.
Make sure you've deployed your changes. Otherwise, please, fix these mistakes, redeploy site and request review again.

**Pull request closed without merging**